extensions/ipynb/package.nls.json

```diff
@@ -6,7 +6,7 @@
 	"newUntitledIpynb.shortTitle": "Jupyter Notebook",
 	"openIpynbInNotebookEditor.title": "Open IPYNB File In Notebook Editor",
 	"cleanInvalidImageAttachment.title": "Clean Invalid Image Attachment Reference",
-	"copyCellOutput.title": "Copy Output",
+	"copyCellOutput.title": "Copy Cell Output",
```

Comment:
As @Tyriar suggested, we could add a Notebook category for the command.

Reply:
that is done in the commands section of package.json

Reply:
<img width="194" alt="image" src="https://github.com/microsoft/vscode/assets/2019016/9b8ac9df-516f-472d-b052-59ab90dd1de2">